portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/APIToken.testcase

```diff
@@ -67,7 +67,8 @@ definition {
 
 		task ("Create a query using the revoked API token") {
 			var curl = '''
-				${analyticsCloudURL}/api/reports/${type} -H "Authorization: Bearer ${token}"
+				${analyticsCloudURL}/api/reports/${type} \
+					-H "Authorization: Bearer ${token}"
```

Comment:
Hey @joshitagaki, I think this test will keep failing because to display the API status (HTTP/1.1 401) you need to use the `-i` argument that you removed in this [other PR](https://github.com/liferay-ac/liferay-portal/pull/607/files) or has this behavior been modified recently?

Reply:
Good catch you're right. I believe the curl still needs to have these changes but it should work with adding the -i argument at the end